packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -666,6 +662,9 @@ describe('StripeWebhookHandler', () => {
       });
 
       it('does not emit a notification for any other subscription state change', async () => {
+        sendSubscriptionUpdatedEmailStub = sandbox
+          .stub(StripeWebhookHandlerInstance, 'sendSubscriptionUpdatedEmail')
+          .resolves({ uid: UID, email: TEST_EMAIL });
```

Comment:
I moved this out of the `beforeEach` hook, since I was getting an error (below) when trying to stub `sendSubscriptionUpdatedEmail` to reject instead of resolve in the new test.
```
 TypeError: Attempted to wrap sendSubscriptionUpdatedEmail which is already wrapped
```
Is there a DRYer way to achieve this?

Reply:
You could run `StripeWebhookHandlerInstance.sendSubscriptionUpdatedEmail.restore()` in a `afterEach`.

Reply:
Thanks for the suggestion -- though I'm still getting the same error with this? That seems like it should work from the [Sinon stub docs](https://sinonjs.org/releases/v11.1.2/stubs/)...

Reply:
Oh I misunderstood and gave a partial suggestion, you could try `sendSubscriptionUpdatedEmail.reset().rejects...`. 🤞

Reply:
I got a way to work: 
```js
beforeEach(() => {
  sendSubscriptionUpdatedEmailStub = sandbox
    .stub(StripeWebhookHandlerInstance, 'sendSubscriptionUpdatedEmail');
});

afterEach(() => {
  StripeWebhookHandlerInstance.sendSubscriptionUpdatedEmail.restore();
});

it('emits a notification when transitioning from "incomplete" to "active/trialing"', async () => {
  sendSubscriptionUpdatedEmailStub.resolves({ uid: UID, email: TEST_EMAIL });
  // ...
```

Reply:
Thanks; your first suggestion combined with `sendSubscriptionUpdatedEmailStub.rejects(fakeAppError);` worked, so I will roll with that, heh.

Reply:
oh sure, not setting the behavior in the beforeEach also works.  I guess there aren't that many places to update to set the behavior in this case.